### ISSUE TITLE
Need useQueue Hook

### ISSUE THREAD

**Issue by aromalanil** [aromalanil/useCustomHooks#36]
## Describe the hook
Since the package already contains a `useStack` hook, it would be great to have a `useQueue` hook.
- This hook will be similar to `useStack` but for Queue data structure.

**PRs are welcome** 🙂

**Comment by MuriloucoLouco** [comment]
I'll take this one.

**Comment by aromalanil** [comment]
@MuriloucoLouco That's great.

## Please ensure these.

- The JSDocs are added to the hook
- There are no linting errors (Run `npm run lint`)
- Documentation is added in docs/readme.md

**Comment by MuriloucoLouco** [comment]
Hi, i ran the linting script, and it returned many errors about linebreaks (678 in total):
```
error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style
error  Delete `?`                                       prettier/prettier
```
I'm new to linting, what is this?

**Comment by aromalanil** [comment]
Okay the repo has a script to fix linting errors automatically.
Run `npm run lint:fix` to automatically fix the majority of linting errors. 
The linting errors occur because this repo follows certain guidelines for code ([Airbnb](https://github.com/airbnb/javascript) specifically).

The errors which are not automatically fixed should be changed manually. This will be relatively easy as linting will mention the error in detail and the line causing the error.

**Comment by MuriloucoLouco** [comment]
Alright, thanks!

**Comment by aromalanil** [comment]
Closed by #37
